cmd/keytransparency-client/cmd/post.go

```diff
@@ -87,7 +89,14 @@ User email MUST match the OAuth account used to authorize the update.
 			return fmt.Errorf("updateKeys() failed: %v", err)
 		}
 		// TODO: fill signers and authorizedKeys.
```

Comment:
Still relevant for authorizedKeys?

Reply:
This is an old comment. Both authorizedKeys and signers are now passed intelligently.

Reply:
Can we remove it then?